### ISSUE TITLE
Tabpy Port issue

### ISSUE THREAD

**Issue by Santosh92Nanduru** [tableau/TabPy#421]
**Environment information:**
 - OS: MacOS Catalina
 - Python version: 3.7

**Describe the issue**
I am unable to establish a link between Tableau and Python using Tabpy. 

I have Anaconda installed in my Macbook. I opened a Jupyter notebook and installed Tabpy using pip install command (screenshot attached).  I tried going into the site-packages and run the .sh file. I did not find the port code(screenshot attached). 

How can I now link Python and Tableau?

![Screenshot 2020-05-28 at 9 36 40 PM](https://user-images.githubusercontent.com/13335676/83165865-b9bc1c00-a12b-11ea-84ad-89c8b341479d.png)
![Screenshot 2020-05-28 at 9 38 41 PM](https://user-images.githubusercontent.com/13335676/83165872-bb85df80-a12b-11ea-9551-130479cb81b0.png)
![Screenshot 2020-05-28 at 9 38 59 PM](https://user-images.githubusercontent.com/13335676/83165880-bcb70c80-a12b-11ea-937b-cbdf3f9dff18.png)

**Comment by 0golovatyi** [comment]
@Santosh92Nanduru You can find instructions for how to install and start TabPy here - https://tableau.github.io/TabPy/docs/server-install.html.

**Comment by Santosh92Nanduru** [comment]
I already installed tabpy server using the pip command in notebook. I still can't establish a connection.

**Comment by 0golovatyi** [comment]
@Santosh92Nanduru Have to try to install TabPy as instructions explain? Are you still experiencing the issue?

**Comment by Santosh92Nanduru** [comment]
Hi. I had followed a different route to address the problem. I entered the admin mode by typing ' sudo su' in the terminal. Once I was there, I installed Tornado 5.1.1. ( I read somewhere this might be the issue). This was strange because I had Tornado 6.1.XXX installed in my laptop. 

However, once this was done, I dragged and dropped the startup.sh file into the terminal. The shell file got executed and I got connected to the port. 

Am not sure how this all works... But this what I precisely did.
